content/resources/operator/_index.md

```diff
@@ -378,3 +378,15 @@ resources:
 	- my_program-TEST1_name3_downloader
 	- my_program-TEST1_name4_downloader
 ```
+
+## 8. ETL and Data Explorer Configurations
+
+ETL stands for “extract, transform, load” and refers to the process of moving data from the PostgreSQL database (graph model) to the ElasticSearch database (flat model), which is carried out by the Gen3 Framework Service [Tube](https://github.com/uc-cdis/tube).
+
+Newly ingested data to the Sheepdog Service can be queried immediately via [Peregrine](https://github.com/uc-cdis/peregrine), but not on the Data Explorer, which is powered by [Guppy](https://github.com/uc-cdis/guppy) on the backend.  During the ETL process, Tube will populate ElasticSearch indices and Guppy makes the ElasticSearch indices available for queries for the Data Explorer.
+
+In practice, Guppy/Tube need to be configured to the ElasticSearch indices in the manifest.json (both in the [versions block](https://github.com/uc-cdis/cdis-manifest/blob/6bc0dc84bec2af7d7971d2a4342319b225728969/gen3.datacommons.io/manifest.json#L23) and the [guppy block for the indices](https://github.com/uc-cdis/guppy#configurations)) and the [ETLmapping.yaml file](https://github.com/uc-cdis/tube/blob/master/docs/configuration_file.md) has to be configured to list those indices.
+Note that configuring the ETLmapping.yaml is dependent on what users want to display on the Explorer page (i.e. File & Data tab) and needs to match to the Data Dictionary. The ETLmapping.yaml can be validated against the Data Dictionary as described [here](https://github.com/uc-cdis/gen3utils#etlmappingyaml-validation).
+After configuring ETLmapping.yaml, indices need to be created, cleaned, or/and re-populated using commands as described [here](https://github.com/uc-cdis/cloud-automation/tree/master/kube/services/tube) or [here](https://github.com/uc-cdis/tube).
```

Comment:
i think https://github.com/uc-cdis/cloud-automation/tree/master/kube/services/tube is outdated, we don't deploy/roll the `tube` service anymore. i think in this sentence you could say something like this instead:
`[...] using the "gen3 gitops configmaps" command to read the new etlMapping.yaml, and the "gen3 job run etl" command to run the ETL`
(or we could update that cloud-automation doc)

And i would remove the 2nd link because i don't see instructions on which commands to run in the `tube` repo